sale_order_ubl/tests/test_ubl_generate.py

```diff
@@ -23,7 +23,7 @@ def test_ubl_generate(self):
                 res = buo.get_xml_files_from_pdf(pdf_file)
                 if order.state in quotation_states:
                     filename = order.get_ubl_filename(
-                        'quotation', version=version)
+                        doc_type='quotation', version=version)
```

Comment:
doc_type is not a named arg in get_ubl_filename()

Reply:
Done. Thanks!